### ISSUE TITLE
How to add a collapsible test step to ExtentTest?

### ISSUE THREAD

**Issue by NiliMe** [anshooarora/extentreports-java#1019]
Hi,
I want to log test steps that are expandable, i.e they have a title and content, and can collapse/expand by click.
I managed to do it when I used sub-nodes, using a new markup.
The code I used:
```
public String getMarkup() {
        return "<ul class="collapsible" data-collapsible="expandable">
                          <li class="active">
                               <div class="collapsible-header active"><i class="material-icons">zoom_in</i>header</div>
                              <div class="collapsible-body" style="display: block;">content</div>
                         </li>
                 </ul>";
}
```

but when I try to log the same markup straight to a "test" (ExtentTest with no sub-node) I can only see the title, and it doesn't expand on click.

This is the expected result:
Step when collapsed:
![image](https://user-images.githubusercontent.com/4667329/35482285-943c7040-043b-11e8-9723-a4a2a1f417eb.png)
Step when opened (clicked):
![image](https://user-images.githubusercontent.com/4667329/35482417-83bbf590-043d-11e8-82c2-8a3fbc8f3fe2.png)

Extent report version: 3.1.0.

Is there a JS/css I can add to make it work, or is it something inside the report infrastructure?
Thanks!

**Comment by anshooarora** [comment]
You can create a toggle by simply doing:

```
test.createNode("Http Response").info(json);
```

**Comment by NiliMe** [comment]
Hi,
I tried what you suggested but it creates a sub-test with these title and content, which come out only in the end of all the other "regular" steps (i.e with title only), and with it's own test-status (unlike step-type: "info").
What I'm trying to create is an actual test-step, same as info/warning etc., which will be exactly where I called it (and not pushed to the end) and is not a test of its own, just a step only with inner content. Is it possible?

Thanks,
Nili.
